addon/components/bs-navbar/toggle.js

```diff
@@ -18,17 +16,19 @@ import deprecateSubclassing from 'ember-bootstrap/utils/deprecate-subclassing';
  * @extends Ember.Component
  * @public
  */
-@tagName('')
 @deprecateSubclassing
 export default class NavbarToggle extends Component {
-  @defaultValue
-  collapsed = true;
+  get collapsed() {
+    return this.args.collapsed ?? true;
+  }
 
   /**
    * @event onClick
    * @public
    */
-  onClick() {}
+  get onClick() {
+    return this.args.onClick ?? (() => {});
+  }
```

Comment:
Can we conditional apply the modifier instead of providing a noop function to it? https://v5.chriskrycho.com/journal/conditional-modifiers-and-helpers-in-emberjs/